invenio_pidstore/providers/datacite.py

```diff
@@ -39,11 +39,11 @@ class DataCite(PidProvider):
     def __init__(self):
         """Initialize provider."""
         self.api = DataCiteMDSClient(
-            username=cfg.get('CFG_DATACITE_USERNAME'),
-            password=cfg.get('CFG_DATACITE_PASSWORD'),
-            prefix=cfg.get('CFG_DATACITE_DOI_PREFIX'),
-            test_mode=cfg.get('CFG_DATACITE_TESTMODE', False),
-            url=cfg.get('CFG_DATACITE_URL')
+            username=current_app.config.get('CFG_DATACITE_USERNAME'),
```

Comment:
Can you change the prefix for these variables from ``CFG_``  to ``PIDSTORE_``. The CFG names are legacy.